lib/jekyll/static_file.rb

```diff
@@ -144,7 +144,11 @@ def copy_file(dest_path)
       if @site.safe || Jekyll.env == "production"
         FileUtils.cp(path, dest_path)
       else
-        FileUtils.copy_entry(path, dest_path)
+        begin
+          FileUtils.ln(path, dest_path)
+        rescue
```

Comment:
I don't actually know Ruby so suggestions appreciated.

As near as I can tell, `FileUtils.ln` simply calls `File.link` which, I believe, is implemented [here](http://rxr.whitequark.org/mri/source/file.c#2436). One of three things happens: (1) `File.link` isn't implemented in which case it raises an exception; (2) it is implemented and `link(2)` fails in which case it raises an exception; or (3) it is implemented and `link(2)` succeeds so it returns 0.

Maybe one could check if `File.link` is implemented ahead of time (I'm not sure if there's a good way or not), but that still doesn't handle `link(2)` being unable to perform the link.